test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java

```diff
@@ -287,8 +282,8 @@ public void close() {
         }
     }
 
-    ControllerStarter startController(int bkPort, int servicePort) throws InterruptedException {
-        return new ControllerStarter(bkPort, servicePort);
+    ControllerStarter startController(int bkPort, int servicePort, int containerCount) throws InterruptedException {
+        return new ControllerStarter(bkPort, servicePort, containerCount);
```

Comment:
Do we need this method ?
why not just call return new SegmentStoreStarter(storageFactory, dataLogFactory, containerCount); directly where it is needed?

Reply:
Done.